_data/speakers.yml

```diff
@@ -388,16 +403,17 @@
   image: GrigoriosMingas_Photo.jpg
   thumbnail: GrigoriosMingas_Photo.jpg
   alt: Grigorios
-  topic:
-  proposal_summary:
+  topic: The MLDA multilevel sampler in PyMC3
+  proposal_summary: This presentation will give you the chance to know more about PyMC3’s new multilevel MCMC sampler, MLDA, and help you use it in practice. MLDA exploits multilevel model hierarchies to improve sampling efficiency compared to standard methods, especially when working with high-dimensional problems where gradients are not available. We will present a step-by-step guide on how to use MLDA within PyMC3, go through its various features and also present some advanced use cases, e.g. employing multilevel PDE-based models written in FEniCS and using adaptive error correction to correct model bias between different levels.
   discourse_link:
   youtube_link:
   description: Dr. Grigorios Mingas is a Senior Research Data Scientist at The Alan Turing Institute.
     He received his PhD from Imperial College London, where he co-designed MCMC algorithms and hardware to accelerate Bayesian inference.
     He has experience in a wide range of projects as a data scientist.
   presentation_type: Talk
-  time_zone:
+  time_zone: Asia/Africa/Europe
   track: Advanced
+  keywork: Grigas
```

Comment:
mlda multilevel sampler has 3 speakers, all should have the same keyword.

Reply:
Done